### ISSUE TITLE
Feature request: option for running complex workflow without fireworks

### ISSUE THREAD

**Issue by JaGeo** [materialsproject/jobflow#302]
I very much like that one can start jobs without setting up a MongoDB database with the help of jobflow. However, for more complex workflows (e.g., the phonon workflow in atomate2), the submission with 1 job script can be very inefficient (e.g., if one computation out of 10 of the workflow does not finish, one has the repeat the overall computation). It would be great to have an obvious and easy to use option in jobflow (e.g., based on a file database) where one can easily restart a part of the computations and/or submit the jobs of an workflow in several job scripts.

I think, @utf , we have talked about this before.

**Comment by arosen93** [comment]
Alex and I were working on an interface to [Covalent](https://github.com/AgnostiqHQ/covalent) that could be used in place of FireWorks as the manager if desired. We are just waiting on an [Covalent bug](https://github.com/AgnostiqHQ/covalent/issues) to be fixed first. Covalent has an optional to (re-)dispatch and monitor jobs locally and without setting up a MongoDB database, so that could be useful when completed.

**Comment by arosen93** [comment]
@JaGeo --- While perhaps not the _perfect_ solution, it sounds like the `MontyStore` in `maggma` is very close in spirit to what you are looking for. See [here](https://github.com/materialsproject/maggma/blob/0e058443d1018b4ab714929c68bf9b9f37f30089/src/maggma/stores/mongolike.py#L879-L895).

**Comment by JaGeo** [comment]
@arosen93 thanks! I now have to find time to test it. Any other suggestions for approaches without db are also highly appreciated. HPCs in Europe and especially their comoute nodes are usually not easy to connect with database servers.

**Comment by utf** [comment]
@JaGeo have you spoken to @davidwaroquiers or @gpetretto about Jobflow-remote?

**Comment by JaGeo** [comment]
Yes, thank you. We have interacted in the meantime. I think we can close this!

**Comment by JaGeo** [comment]
Hi @utf @davidwaroquiers , 

I wanted to raise this point again. It might be worth considering a middle ground solution for smaller workflows, potentially with another type of store. jf-remote is awesome but still requires some setup.

**Comment by Andrew-S-Rosen** [comment]
This feature request is likely partially dependent on https://github.com/materialsproject/maggma/issues/832.

**Comment by JaGeo** [comment]
@Andrew-S-Rosen thanks for linking this!
